CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.3.1.2 - 2024-09-06
```

Comment:
Due to how the versioning works, this will actually be version 1.3.2 (admittedly that's not immediately obvious). Can you update that please, and probably change the date to today (so it will align with when the package is published)?

Reply:
updated version to 1.3.2 and amended the change log as well

Reply:
Ah sorry, probably should have been clearer. The version will automatically be incremented based on the latest version of the package that's published (which is actually version 1.3.3). Can you revert the .csproj changes and update the changelog to version 1.3.4 (as that will be the next version published)?

Reply:
No worries, reverted proj changes and set the change log to 1.3.4